chall02/rjaakonm.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int		ft_int_clamp(int nb, int min, int max)
+{
+	if (nb < min)
+		return (min);
+	if (nb > max)
+		return (max);
+	return (nb);
+}
+
+char *hv_rgb2hex(int r, int g, int b)
+{
+	int				rgb[3];
+	unsigned int 	color;
+	char			*color_string;
+
+	if (!(color_string = (char *)malloc(sizeof(char) * 8)))
+		return NULL;
+
+	rgb[0] = ft_int_clamp(r, 0, 255);
+	rgb[1] = ft_int_clamp(g, 0, 255);
+	rgb[2] = ft_int_clamp(b, 0, 255);
+	color = rgb[0] << 16 | rgb[1] << 8 | rgb[2];
```

Comment:
bit wise :) where did you find this idea?

Reply:
I've been using it for all graphic branch stuff, was the fastest and shortest way I found to do those, but yeah wasn't useful for this since I still used sprintf :p but works great if I need to manipulate colors!